src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -145,6 +145,12 @@ def go_from_trigger_mozilla_central(self):
         )
         logger.info('Report generated successfully')
 
+        report = json.loads(output)
+        expected_extensions = ['.js', '.cpp']
+        for extension in expected_extensions:
+            assert any(file['name'].endswith(extension) for file in
```

Comment:
Nit: Use `f` instead of `file`, as `file` is a reserved keyword.

Reply:
Oh yes. Sorry I should've taken care of that.